cloud/src/meta-service/meta_service.cpp

```diff
@@ -406,7 +406,7 @@ void MetaServiceImpl::batch_get_version(::google::protobuf::RpcController* contr
 void internal_create_tablet(MetaServiceCode& code, std::string& msg,
                             const doris::TabletMetaCloudPB& meta, std::shared_ptr<TxnKv> txn_kv,
                             const std::string& instance_id,
-                            std::set<std::pair<int64_t, int32_t>>& saved_schema) {
+                            std::set<std::pair<int64_t, int32_t>>& saved_schema, int64_t db_id) {
```

Comment:
> MS may serve an older version of FE, so db_id has not been set. It would be more appropriate to use `std::optional<int64_t>` here and check `request.has_db_id()` when calling.

resolved, i not use std::optional instead of using request